doc/integrator/backend_qgis.rst

```diff
@@ -76,10 +68,17 @@ and add a section for the DB you want to access:
 
    [<package>]
    host=<host>  # Localhost address of the Docker network interface (=> ${IP})
-   dbname=<database name>
+   port=5432
```

Comment:
why not write
<database port>
?

Reply:
? what's the relation with write?

Reply:
Hmm something disappeared in my comment.
I tried to say: 

why not write
  database port
instead of 5432.

Reply:
It's funny, if you edit my earlier comment, you see what I wrote. But GitHub does not display it. I guess the smaller than and greater than characters are somehow special in comments.

Reply:
Effectively I can change that :-)